cluster/ec2_banzaicloud.go

```diff
@@ -206,8 +207,20 @@ func (c *EC2ClusterBanzaiCloudDistribution) DownloadK8sConfig() ([]byte, error)
 }
 
 func (c *EC2ClusterBanzaiCloudDistribution) GetAPIEndpoint() (string, error) {
```

Comment:
Add some newlines to increase the readability of this function. (Thumb rule: add newlines before and after control structures (except in case of error handling), add newlines before returns, otherwise separate code into groups which logically belong together (eg. variable declarations)